hazelcast/src/test/java/com/hazelcast/map/BasicMapTest.java

```diff
@@ -1191,6 +1194,30 @@ public void testIfWeCarryRecordVersionInfoToReplicas() {
     }
 
     @Test
+    public void github_11489() throws Exception {
+        IMap test = getInstance().getMap("github_11489");
```

Comment:
Should be `IMap<Integer, Integer> test = getInstance().getMap("github_11489");`, no need to generate generics warnings.